src/overlays/actors/ovl_En_And/z_en_and.c

```diff
@@ -30,22 +29,158 @@ ActorInit En_And_InitVars = {
     (ActorFunc)EnAnd_Draw,
 };
 
-#endif
-
-extern UNK_TYPE D_0600B380;
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_And/func_80C18B90.s")
+static AnimationInfoS sAnimationInfo[8] = {
+    { &gAndStaticAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &gAndIdleAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &gAndWalkAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &gAndRaiseHeadAnim, 1.0f, 0, -1, ANIMMODE_ONCE, 0 },
+    { &gAndRaisedHeadLoopAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &gAndRaiseHandAnim, 1.0f, 0, -1, ANIMMODE_ONCE, 0 },
+    { &gAndRaisedHandLoopAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &gAndRaisedHandWalkAnim, 1.0f, 0, -1, ANIMMODE_ONCE, 0 },
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_And/func_80C18BD8.s")
+static TexturePtr sEyeTextures[4];
```

Comment:
This almost looks like bss (though is actually just a forward declare). It would probably be better just to put all the data up here instead.

Reply:
Hi, thanks for the feedback! This particular static is physically located after the array in `HandleCsActions`, so it does not match if the data is defined before it. I added the forward declare after it was suggested I use `ARRAY_COUNT` in `Blink`.

Reply:
```suggestion
extern TexturePtr sEyeTextures[4];
```
Ah, must have missed that. Would it work as an extern instead?

Reply:
Tried that too, unfortunately it won't compile
```
src/overlays/actors/ovl_En_And/z_en_and.c:162:19: error: static declaration of 'sEyeTextures' follows non-static declaration
  162 | static TexturePtr sEyeTextures[4] = {
      |                   ^~~~~~~~~~~~
src/overlays/actors/ovl_En_And/z_en_and.c:41:19: note: previous declaration of 'sEyeTextures' with type 'void *[4]'
   41 | extern TexturePtr sEyeTextures[4];
      |                   ^~~~~~~~~~~~
```

Reply:
In that case I think it would be better to have a define instead like `EYE_TEXTURES_COUNT` and use it in the array and the `EnAnd_Blink`, because this forward declare looks a bit confusing at first glance
```c
#define EYE_TEXTURES_COUNT 4
```